x-pack/plugins/reporting/server/config/create_config.test.ts

```diff
@@ -145,10 +140,49 @@ describe('Reporting server createConfig$', () => {
         encryptionKey: '888888888888888888888888888888888',
       })
     );
-    const mockConfig$: any = mockInitContext.config.create();
+    const mockConfig$ = createMockConfig(mockInitContext);
     const result = await createConfig$(mockCoreSetup, mockConfig$, mockLogger).toPromise();
 
     expect(result.capture.browser.chromium).toMatchObject({ disableSandbox: expect.any(Boolean) });
-    expect((mockLogger.warn as any).mock.calls.length).toBe(0);
+    expect(mockLogger.warn.mock.calls.length).toBe(0);
   });
+
+  for (const hostname of [
+    '0',
+    '0.0',
+    '0.0.0',
+    '0.0.0.0',
+    '0000:0000:0000:0000:0000:0000:0000:0000',
+    '::',
+  ]) {
+    it(`apply failover logic when hostname is given as ${hostname}`, async () => {
```

Comment:
Jest provides tooling for such cases. Please use `it.each` instead.

```suggestion
  it.each`
    hostname
    ${'0'}
    ${'0.0'}
    ${'0.0.0'}
    ${'0.0.0.0'}
    ${'0000:0000:0000:0000:0000:0000:0000:0000'}
    ${'::'}
  `('apply failover logic when hostname is given as $hostname', async ({ hostname }) => {
```